helm/examples/values-local-cluster-ogdc-example.yaml

```diff
@@ -19,17 +19,10 @@ global:
   ## (Note the actual key passwordsSecret is not used anywhere; only the yaml anchor/alias
   ##  &passwordSecretName. IF YOU OVERRIDE THIS VALUE IN ANOTHER VALUES-OVERLAY FILE, YOU ALSO
   ## NEED TO OVERRIDE THE VALUES THAT USE THIS ANCHOR - search this file for 'passwordSecretName')
-  ##
-  ## 
-  ## 1. Define the release name on the command line:
-  ##     $  RELEASE_NAME=my-release  envsubst < helm/examples/values-dev-cluster-ogdc-example.yaml | kubectl apply -n <mynamespace> -f -
-  ##
-  ##     # if this results in "command not found: envsubst", you can install envsubst (e.g. on Mac OS:
-  ##        $  brew install gettext && brew link --force gettext
-  ##
-  ## 2. Alternatively, you can simply replace "${RELEASE_NAME}" in the file below 
-  ##
-  passwordsSecret: &passwordSecretName ${RELEASE_NAME}-secrets
+  ## NOTE: in local development, the release name is expected to be
+  ## `qgnet-ogdc`. If this changes, the value here should be updated:
+  ## `{RELEASE_NAME}-secrets`.
+  passwordsSecret: &passwordSecretName qgnet-ogdc-secrets
```

Comment:
The use of `${RELEASE_NAME}` in the values yaml breaks `skaffold`, so I just replaced it with the value we use. In local dev, we can expect a consistent release name.

I would love to address this in a different way, but it seems like a generally unsolved problem with `helm` (see: https://github.com/helm/helm/issues/12323). [helmfile](https://github.com/helmfile/helmfile) seems like a common tool for managing this kind of thing, but I don't think we need to dive into that right now.